data/sql/updates/pending_db_world/rev_1626532264671526700.sql

```diff
@@ -0,0 +1,10 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1626532264671526700');
+
+-- We had 2 gossips with the id, and it was choosing the wrong one. So we insert into a new one id.
+DELETE FROM gossip_menu WHERE menuid = 61025 and textid = 10041;
```

Comment:
Please also add backticks ` around column names

Reply:
Added the backticks